packages/botonic-plugin-flow-builder/src/api.ts

```diff
@@ -0,0 +1,162 @@
+import { Input } from '@botonic/core'
+import axios from 'axios'
+
+import {
+  HtFallbackNode,
+  HtFlowBuilderData,
+  HtIntentNode,
+  HtKeywordNode,
+  HtNodeComponent,
+  HtNodeStartType,
+  HtNodeWithContent,
+  HtNodeWithContentType,
+  HtStartNode,
+} from './content-fields/hubtype-fields'
+import { FlowBuilderApiOptions } from './types'
+
+export class FlowBuilderApi {
+  private readonly url: string
+  public flow: HtFlowBuilderData
+
+  constructor(options: FlowBuilderApiOptions) {
+    this.url = options.url
+    if (options.flow) {
+      this.flow = options.flow
+    }
+  }
+
+  async init(token: string) {
```

Comment:
I think this name can be more descriptive like `initializeFlow()` or something like that?

Reply:
I've refactored the way the `FlowBuilderApi` is initialized, now you have to call a static method `create` that will create a new `FlowBuilderApi` with the flow already fetched.